src/transcoder/transcoder.cpp

```diff
@@ -195,10 +195,16 @@ void Transcoder::JobState::PostFinished(bool success) {
                               new Transcoder::JobFinishedEvent(this, success));
 }
 
+QString Transcoder::JobState::GetDisplayName() {
+  return QFileInfo(job_.input).fileName() + " to " +
```

Comment:
If you use an arrow here, (`->` perhaps) then we don't need to worry about translations.

Reply:
Changed to use `=>`.